features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml

```diff
@@ -3,6 +3,11 @@ x-git-defaults: &git_defaults
     tag: branch-24.08
     upstream: rapidsai
 
+x-rapids-build-backend-args: &rapids_build_backend_args |
+  --config-settings skbuild.strict-config=false
+  --config-settings rapidsai.disable-cuda=true
+  --config-settings rapidsai.matrix_entry=cuda=$(grep -o '^[0-9]*.[0-9]*' <<< "${CUDA_VERSION}")
```

Comment:
```suggestion
  --config-settings rapidsai.matrix_entry=cuda=$(grep -o '^[0-9]+.[0-9]+' <<< "${CUDA_VERSION}")
```

Very very minor thing, but I think we want `+` here so this will fail if `CUDA_VERSION` accidentally contains something like `"."`.

```bash
grep -o '^[0-9]*.[0-9]*' <<< "."
# .

echo $?
# 0

grep -o '^[0-9]+.[0-9]+' <<< "."
# (empty)

echo $?
# 1
```